target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -190,10 +190,19 @@ TARGET_DEVICES += ubnt_litebeam-ac-gen2
 define Device/ubnt_nanobeam-ac
   $(Device/ubnt-wa)
   DEVICE_MODEL := NanoBeam AC
+  DEVICE_VARIANT := Gen1
   DEVICE_PACKAGES += kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct rssileds
 endef
 TARGET_DEVICES += ubnt_nanobeam-ac
 
+define Device/ubnt_nanobeam-ac-gen2
+  $(Device/ubnt-wa)
+  DEVICE_MODEL := NanoBeam AC
+  DEVICE_VARIANT := Gen2
+  DEVICE_PACKAGES += kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct rssileds
```

Comment:
As stated above: Do you actually have rssileds?

Reply:
Just for the record: Why does nobody ever answer a question in this place?

So, now, I have to go through all these closed conversations, open all of them manually, and find out what the answer to my question could be by comparing code.

Reply:
Yes. It has rssi leds. I have added the code that defines the rssileds from other ar9342 devices. @ae6xe Could u test the changes? Otherwise I have to climb on a roof again.

Reply:
@PolynomialDivision , yes will test today on NanoBeam AC gen2 and Nanostation AC.

@adschm re: "Do you actually have rssileds".   Both of these devices have blue leds for 1) Power; 2) main; 3) secondary/LAN2; 4) multiple LEDs for RSSI.

Reply:
@PolynomialDivision this commit has conflicts to merge with master. Rebase pull with master, so I can build, then test?